bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/IPageLayout.java

```diff
@@ -577,4 +577,30 @@ public interface IPageLayout {
 	 * @since 3.3
 	 */
 	IPlaceholderFolderLayout getFolderForView(String id);
+
+	/**
+	 * Sets the onboarding text. The text is shown in the editor area in case no
+	 * editor is open.
+	 *
+	 * @param text the onboarding text
+	 * @since 3.129
+	 */
+	void setEditorOnboardingText(String text);
+
+	/**
+	 * Sets the onboarding image. The image is shown in the editor area in case no
+	 * editor is open.
+	 *
+	 * @param imageUri the uri of the onboarding image
+	 * @since 3.129
+	 */
+	void setEditorOnboardingImage(String imageUri);
```

Comment:
"setEditorOnboardingImageUri" seems a better name. Because you don't set an Image, but "only" it's Uri.